release/src/main/scripts/build_release_candidate.sh

```diff
@@ -90,6 +91,14 @@ if [[ $confirmation != "y" ]]; then
   exit
 fi
 
+RC_TAG="v${RELEASE}-RC${RC_NUM}"
+echo "Pushing tagged commit for RC${RC_NUM}"
+sh $SCRIPT_DIR/set_version.sh ${RELEASE} --release"
```

Comment:
So we run this for every RC but with the same state ?

Reply:
Yes. The release branch still has SNAPSHOT version and for each RC we peel off a separate commit with the version set to non-SNAPSHOT. My other PR actually moves this logic out of this script.